contracts/cheddar_5x5_tic_tac_toe/src/game.rs

```diff
@@ -159,7 +159,9 @@ impl Game {
             self.current_player_account_id(),
             "Can't claim timeout win if it's your turn"
         );
-        //3. Check for timeout
+        //3. Check if the player invoking the method is in the game
+        assert!(self.contains_player_account_id(&player), "No access");
```

Comment:
so there was a way that someone else will do a move instead of us?
Good catch!

Could you also add a test for this case please?